doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-kogito-cli.adoc

```diff
@@ -0,0 +1,88 @@
+[id="proc-kogito-microservices-deploying-on-ocp-kogito-cli_{context}"]
+= Deploying {KOGITO} microservices on OpenShift using the {KOGITO} CLI
+
+The {KOGITO} command-line interface (CLI) enables you to interact with the {KOGITO} Operator for deployment tasks. The {KOGITO} CLI also enables you to deploy {KOGITO} microservices from source instead of relying on custom resources and YAML files. You can use the {KOGITO} CLI as a command-line alternative for deploying {KOGITO} microservices without the {OPENSHIFT} web console.
+
+.Prerequisites
+* The `oc` OpenShift CLI is installed and you are logged in to the relevant OpenShift cluster. For `oc` installation and login instructions, see the https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/cli_tools/openshift-cli-oc[OpenShift documentation].
+* You have OpenShift permissions to create resources in a specified namespace.
+* (Quarkus only) The `pom.xml` file of your {PRODUCT} project contains the following dependency for the https://quarkus.io/guides/microprofile-health[Quarkus `smallrye-health` extension]. This extension enables the https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes[liveness and readiness probes] that are required for Quarkus-based projects on OpenShift.
++
+.SmallRye Heath dependency for Quarkus applications on OpenShift
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-smallrye-health</artifactId>
+</dependency>
+----
+
+.Procedure
+. Go to the https://github.com/kiegroup/kogito-operator/releases[`kogito-operator`] releases page in GitHub and download the latest version of the `kogito-cli-_RELEASE_` binary file that is specific to your operating system.
```

Comment:
`https://github.com/kiegroup/kogito-operator/releases` is a release page for community kogito operator.